task/orchestrator.go

```diff
@@ -249,7 +163,7 @@ func (o *Orchestrator) handleRunErrors(ctx context.Context, err error) error {
 	return errors.Join(failErr, unclaimErr, err)
 }
 
-var reapTime = 10 * time.Second // can be overridden by tests
+var reapTime = 30 * time.Second // can be overridden by tests
```

Comment:
Extended this more to match the default grace period. It's non-blocking, anyway.